README.md

```diff
@@ -0,0 +1,15 @@
+# dvbtee-express
+This is a basic example of how to use node-dvbtee with udp socket.
```

Comment:
I'd be happy to merge your code, but remember that the `README.md` is for the entire repository.  Can you change "with udp socket" to something more like "using various interfaces / methods" or something else that you think may be more appropriate.